modules/nw-egress-ips-automatic.adoc

```diff
@@ -10,8 +10,9 @@ for a specific namespace across one or more nodes.
 
 .Prerequisites
 
-* Install the OpenShift CLI (`oc`).
-* Access to the cluster as a user with the `cluster-admin` role.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* Your project has an automatically generated `NetNamespace` object. (Creating your own will lead to errors.)
```

Comment:
This doesn't feel like the right way of explaining this to me. I think it would make more sense that the "Procedure" section where it says "Update the NetNamespace object ..." to clarify that you have to wait for the system to create the NetNamespace object, and cannot create it yourself.

Reply:
+1 that would be my suggestion as well.

Reply:
Thanks, Dan. I agree -- it doesn't feel entirely right. It's.not the cleanest way to use a prerequisite.

That said, here's my thinking:

The procedure assumes a certain beginning state, which includes a system-generated NetNamespace object. If your beginning state includes one you created yourself, stuff breaks.

In this version, we establish this state in the right sequence, before step 1's "Update the NetNamespace object".

In any other version, we confuse and frustrate users by getting the sequence out of order. Users get started with the procedure, only to be told "By the way, if you tried to make one yourself, you'll need to undo that now."

The situation is made more difficult by the specific way step 1 is written, which doesn't lend itself to easy interjections or asides. Adding a statement anywhere but the end seriously disrupts the flow. 

I'm adding a commit with the note-type fix on one of the procedures.

Take a look, compare, and let me know what you think.

Reply:
> The procedure assumes a certain beginning state, which includes a system-generated NetNamespace object.

Yeah, but like, we assume the server isn't on fire too. Having a system-generated NetNamespace is just _normal_, so it feels weird to call it a "prerequisite"...

Reply:
@danwinship 

Agreed -- it's a weird thing to document, and it's messy in either location.

Here's an update with a note in the procedure. How's that look to you?

[procedure 1](https://deploy-preview-32639--osdocs.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/assigning-egress-ips.html#nw-egress-ips-automatic_egress-ips)
[procedure 2](https://deploy-preview-32639--osdocs.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/assigning-egress-ips.html#nw-egress-ips-static_egress-ips)